frontend/package.json

```diff
@@ -33,7 +33,10 @@
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
     "test:local": "jest --watchAll",
-    "test:type-check": "tsc --noEmit"
+    "test:type-check": "tsc --noEmit",
+    "test:storybook": "test-storybook --verbose",
```

Comment:
Interesting -- this doesn't seem to fail on a11y issues -- shouldn't it?

Reply:
right now no. I can add that though. I didnt know if this was the way we wanted to test a11y yet

Reply:
Anyway to get additional testing blocks sounds like a plan to me.

Reply:
I will add an update commit with that implementation